rest_framework/renderers.py

```diff
@@ -260,88 +268,93 @@ class HTMLFormRenderer(BaseRenderer):
     base_template = 'form.html'
 
     default_style = ClassLookupDict({
-        serializers.Field: {
+        Field: {
```

Comment:
Agreed. In a new ticket we will use `relations.` and `fields.` to replace where `serializers.` used to be used (since those fields aren't actually in `serializers`, they were just available because of the `import *`)